src/main/java/org/zone/ZonePlugin.java

```diff
@@ -69,15 +69,45 @@
 @Plugin("zones")
 public class ZonePlugin {
 
+    /**
+     * Plugin container of this plugin
+     */
     private final PluginContainer plugin;
+    /**
+     * Logger of this plugin
```

Comment:
only needed to do the things that were public or protected, but i like the style

Reply:
Lol